Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,41 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string name = "Hancock";
+            string surname = "Johnson";
+            int age = 20;
+            float weight = 81;
+            float heightcm = 180;
+            float heightm = heightcm / 100;
+            float heightsq = heightm * heightm;
+            float BMI = weight / heightsq;
+
+            Console.WriteLine("Name: " + name);
+            Console.WriteLine("Surname: " + surname);
+            Console.WriteLine("Age: " + age);
+            Console.WriteLine("Weight: " + weight + "kg");
+            Console.WriteLine("Height: " + heightcm + "cm");
+            Console.WriteLine("BMI: " + BMI);
+
+            Console.WriteLine();
+
+            string name2 = "Debra";
+            string surname2 = "Jones";
+            int age2 = 22;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add the 'const' modifier to 'age2'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4836333)